### ISSUE TITLE
possible to release the sample code?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jupe/gppanel#5]
```
Hi, I do not see the sample code of using gppanel in the Git repo, I suggest 
you can adding one target about the sample code in the gpPanel.cbp.

Thanks.


```

Original issue reported on code.google.com by `Asmwarr...@gmail.com` on 14 Jun 2013 at 8:54

**Comment by shaiku** [comment]
Apparently the sample source was lost in a hard drive crash so it will not be available.
I made a new basic Visual Studio example in my fork.  Perhaps you could look at that or look at this one: https://github.com/Shawn666-Liu/gppanel/blob/master/Sample/main.cpp

**Comment by jupe** [comment]
this example is now merged here as well (#9).
